db/db.go

```diff
@@ -134,6 +140,50 @@ func (db *DB) Close() error {
 	return db.sqlite3conn.Close()
 }
 
+// Destroy drops all the tables from the database. This is useful for resetting
+// an in-memory database when necessary.
+func (db *DB) Destroy() error {
```

Comment:
Is there no way to do this by re-opening the database connection? Such that you get a new lump of RAM?